### ISSUE TITLE
Inquiry About VoxBlink2 Training Configuration and Hyperparameters

### ISSUE THREAD

**Issue by coutAfaiendl** [PalabraAI/redimnet2#4]
I would like to ask whether you have conducted training experiments on VoxBlink2. If so, would it be possible to share the corresponding hyperparameters and experimental configuration?

In particular, I am interested in details such as the model architecture, training setup, data augmentation strategy, optimizer and learning rate schedule, margin settings, batch size, and any other important training parameters.

**Comment by vanIvan** [comment]
> I would like to ask whether you have conducted training experiments on VoxBlink2.

Yes we did, there is a single model trained on voxblink2+voxceleb2 datasets:
```
# VoxBlink2 + VoxCeleb2 variant (stronger, B6 only)
model = torch.hub.load("PalabraAI/redimnet2", "redimnet2",
                      model_name="b6", train_type="lm",
                      dataset="vb2+vox2_v0", pretrained=True)
```
> If so, would it be possible to share the corresponding hyperparameters and experimental configuration?

As I alredy [replied](https://github.com/IDRnD/redimnet/issues/29) in redimnet original repo: we are planning to release training code and recipes there in coming months
